package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@oat-sa/tao-item-runner-qti",
-    "version": "0.20.0",
+    "version": "0.21.0",
```

Comment:
**chore:** this is not needed anymore. The versioning will be managed by the release tool.

Reply:
removed!